From: Liang Yan <lyan@suse.com>
Subject: kABI workaround for struct kvm_vcpu_arch
Patch-mainline: Never, kABI workaround
References: git-fixes

Add a new field to  by the patch 
patches.suse/KVM-arm64-Support-stolen-time-reporting-via-shared-s.patch

Signed-off-by: Liang Yan <lyan@suse.com>
---
 arch/arm64/include/asm/kvm_host.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/include/asm/kvm_host.h
+++ b/arch/arm64/include/asm/kvm_host.h
@@ -340,12 +340,14 @@ struct kvm_vcpu_arch {
 	 * see kvm_vcpu_load_sysregs and kvm_vcpu_put_sysregs. */
 	bool sysregs_loaded_on_cpu;
 
+#ifndef __GENKSYMS__
 	/* Guest PV state */
 	struct {
 		u64 steal;
 		u64 last_steal;
 		gpa_t base;
 	} steal;
+#endif
 };
 
 /* Pointer to the vcpu's SVE FFR for sve_{save,load}_state() */
